build.sbt

```diff
@@ -1,56 +1,63 @@
 name := "IterablePlayUtils"
+organization := "com.iterable"
+homepage := Some(url("https://github.com/Iterable/iterable-play-utils"))
+scmInfo := Some(
+  ScmInfo(url("https://github.com/Iterable/iterable-play-utils"), "scm:git@github.com:Iterable/iterable-play-utils.git")
+)
 
-version := "1.1.1"
+licenses := Seq("Apache-2.0" -> url("http://www.apache.org/licenses/LICENSE-2.0.txt"))
```

Comment:
LICENSE is Apache already. I didn't realize the license was already here when I added LICENSE to this repo.

Reply:
Oh. You added the Apache License earlier, I missed that. Should we switch back to MIT?

Reply:
I don't have a strong preference either way, but I tend to default to Apache because it provides a patent grant and is more explicit about what you can do.